sdks/v4-sdk/src/utils/v4Planner.ts

```diff
@@ -17,27 +17,26 @@ export enum Actions {
   DECREASE_LIQUIDITY = 0x01,
   MINT_POSITION = 0x02,
   BURN_POSITION = 0x03,
+
```

Comment:
i wonder if we should put in placeholder comments about why some numbers are missing?

Reply:
yeah will do!